bundles/binding/org.openhab.binding.bticino/src/main/java/org/openhab/binding/bticino/internal/BticinoDevice.java

```diff
@@ -174,12 +213,155 @@ public void receiveCommand(String itemName, Command command, BticinoBindingConfi
+                                    logger.debug("RollershutterPosEnd %: {}: {}=>{} / {}", itemName,
+                                            RollerShutterPosStart, RollerShutterPosEnd,
+                                            Integer.toString(RollerShutterPosEnd));
+                                    ShutterBlock.put(l_where, false);
+                                    return;
+                                } else if (ShutterBlock.getOrDefault(l_where, false) == false) {
+                                    logger.debug("RollershutterPosStart: {}: {}", itemName, RollerShutterPosStart);
+                                    if (UpDownType.UP.equals(command)) {
+                                        l_pr.addProperty("what", "1");
+                                        RollerShutterTimeStart.put(l_where, new Date((new Date()).getTime()));
+                                        RollerShutterDir.put(l_where, "UP");
+                                    } else if (UpDownType.DOWN.equals(command)) {
+                                        l_pr.addProperty("what", "2");
+                                        RollerShutterTimeStart.put(l_where, new Date((new Date()).getTime()));
+                                        RollerShutterDir.put(l_where, "DOWN");
+                                    } else if (StopMoveType.STOP.equals(command)) {
+                                        l_pr.addProperty("what", "0");
+                                        m_open_web_net.onCommand(l_pr);
+                                        if (RollerShutterDir.getOrDefault(l_where, "") != "") {
+                                            if (RollerShutterDir.getOrDefault(l_where, "") == "UP") {
+                                                logger.debug("RollerShutterTimeStart: {}: {}", itemName,
+                                                        ((new Date()).getTime() - RollerShutterTimeStart
+                                                                .getOrDefault(l_where, new Date((new Date()).getTime()))
+                                                                .getTime()));
+                                                Long RollerShutterPosEnd = ((new Date()).getTime()
+                                                        - RollerShutterTimeStart
+                                                                .getOrDefault(l_where, new Date((new Date()).getTime()))
+                                                                .getTime())
+                                                        * 100 / m_ShutterRunTime;
+                                                RollerShutterPos.put(l_where,
+                                                        (int) (RollerShutterPosStart - RollerShutterPosEnd));
+                                            }
+                                            if (RollerShutterDir.getOrDefault(l_where, "") == "DOWN") {
```

Comment:
Need to use `.equals()` and not `==`

Reply:
OK
see commit https://github.com/openhab/openhab1-addons/pull/5486/commits/7b22fc948ebf70c408e256e9476b5ee061a7e725